deploy-sandpit.sh

```diff
@@ -7,7 +7,7 @@ function runTerraform() {
   echo "Running ${1} Terraform..."
   pushd "${DIR}/ci/terraform/${1}" >/dev/null
   rm -rf .terraform/
-  terraform init -backend-config=sandpit.hcl
+  terraform init -upgrade -backend-config=sandpit.hcl
```

Comment:
yeah, otherwise peoples' local machines will have deploy problems and it's hard to fix without putting it inside the script. we don't commit the `.terraform.lock.hcl` - if we did, this wouldn't be a good idea to keep